client/src/components/TopSongGallary/TopSongGallary.jsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Box,
+  Image,
+  ImageBox,
+  ImageBoxContainer,
+  SongGallary,
+  SongP,
+  Songartist,
+  Title,
+  TopASongCountainer,
+  TopHSongCountainer,
+} from "./TopSongGallaryStyle";
+import cover1 from "../../assets/TopSongGallary/Rectangle 3.png";
+import cover2 from "../../assets/TopSongGallary/Rectangle 2.png";
+import cover4 from "../../assets/TopSongGallary/Rectangle 1.png";
+import cover5 from "../../assets/TopSongGallary/Rectangle 4.png";
+
+export default function TopSongGallary() {
+  return (
+    <SongGallary>
+      <TopHSongCountainer>
+        <Title>Top Hebrow Songs</Title>
+        <ImageBoxContainer>
```

Comment:
export IMageBoxContaner to a new component, and pass the img, artist and song name as props. you have a  lot of DRY because of this

Reply:
done